tests/e2e/deploy_enable_disable_test.go

```diff
@@ -0,0 +1,131 @@
+package e2e_test
+
+import (
+	"github.com/kyma-project/cli/internal/cli"
+	. "github.com/kyma-project/cli/tests/e2e"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+// TODO: Drop Is from function names (add in the middle)
```

Comment:
what does this TODO means?

Reply:
forgot to remove it :D was one of Aleksandr's comments for the function naming